src/app/forms/geoff-moores/geoff-moores.jade

```diff
@@ -1,4 +1,4 @@
-.form
+.form(ng-app="uvp-builder-web.forms")
```

Comment:
add `uvp-builder-web.forms ` to the list of dependencies of `uvp-builder-web`, so you don't need to specify the app on each view